DSharpPlus.Rest/DiscordRestClient.cs

```diff
@@ -1215,7 +1215,7 @@ public Task DeleteWebhookAsync(ulong webhook_id, string reason, string webhook_t
         /// <param name="builder">Webhook builder filled with data to send.</param>
         /// <returns></returns>
         public Task<DiscordMessage> ExecuteWebhookAsync(ulong webhook_id, string webhook_token, DiscordWebhookBuilder builder)
-            => this.ApiClient.ExecuteWebhookAsync(webhook_id, webhook_token, builder.Content, builder.Username, builder.AvatarUrl, builder.IsTTS, builder.Embeds, builder.Files, builder.Mentions);
+            => this.ApiClient.ExecuteWebhookAsync(webhook_id, webhook_token, builder.Content, builder.Username, builder.AvatarUrl, builder.IsTTS, builder.Embeds, builder._files, builder.Mentions);
```

Comment:
Small nit here: Perhaps change this back to `builder.Files` for consistency.

Reply:
This is actually probably gonna change anyway when i do my next pr which is a cleanup for the webhook builder and we are gonna pass the builder the whole way through.  So if u want, we can wait for this request change till then

Reply:
After talking with @Neuheit in discord, this is gonna wait